package.json

```diff
@@ -70,6 +70,7 @@
     "@types/express": "^4.17.2",
     "@types/jest": "^29.4.0",
     "@types/jsdom": "^12.2.2",
+    "@types/marked": "^5.0.0",
```

Comment:
I chose Marked because it is the most popular regularly maintained Markdown to HTML converter NPM package I could find.